x-pack/platform/plugins/shared/alerting/server/application/rule/types/rule.ts

```diff
@@ -85,6 +85,7 @@ export interface Rule<Params extends RuleParams = never> {
   alertDelay?: RuleSchemaType['alertDelay'];
   legacyId?: RuleSchemaType['legacyId'];
   flapping?: RuleSchemaType['flapping'];
+  artifacts?: RuleSchemaType['artifacts']; // do I need to remove it from here?
```

Comment:
Just adding a comment here so that @cnasikas sees this question.

Reply:
This is a leftover comment, which I removed in this [commit](d39cfceeabe390ca435a1ad79698f9a161a58a82).

We have been discussing about the requirement of not returning artifacts in the rule response. The place where we don't need to return artifacts is in the last [transformation_rule_to_rule_response ](x-pack/platform/plugins/shared/alerting/server/routes/rule/transforms/transform_rule_to_rule_response/v1.ts)step